### PR TITLE
Fix log policy and reword admin api introduction

### LINKED ISSUE #190: Docs issues
* `log` policy is called `simple-logger`, which is wrong.
* `log` policy does not tell that message is printed using `INFO` as log level.
* The Admin API needs an introduction paragraph explaining what's going on, why do you need it and what you can do with that

### PR DESCRIPTION
**Pull request by XVincentX**
Connect #190 
Closes #190

### CODE REVIEW

File: docs/admin/index.md [hunk 1]
```diff
@@ -6,51 +6,17 @@ doc-order: 10.0
...
+This API is designed for internal use and provides a certain degree of control over Express-Gateway, so its public
+exposure is not usually a great idea. If you really have to, you should really make sure it is properly secured. You can
+find one way later in this document.
 
 Note: in the future the Admin API will also allow configuration of gateway entities such as pipeline and policy
 configuration dynamically.
 
 The Express Gateway [CLI][cli] utilizes the Admin API.
 
-### Core Entities
```
Comment (DrMegavolt) [inline docs/admin/index.md]: why removed ?

Comment (XVincentX) [reply docs/admin/index.md]: The paragraph I removed is a trimmed down (and old) version of what we have here: https://www.express-gateway.io/docs/consumer-management and couple of other links.

I'd rather add a link saying "To get more information on core entities, please see …, …, …)

File: docs/admin/index.md [hunk 2]
```diff
@@ -6,51 +6,17 @@ doc-order: 10.0
...
 
-The Express Gateway Admin API is an HTTP interface to manage internal entities such as Users, Applications, Credenitials,
-Scopes, OAuth2 tokens and the relationships among such entities.
+The Express Gateway Admin API is an HTTP interface for administration purposes.
+
+This API is designed for internal use and provides a certain degree of control over Express-Gateway, so its public
+exposure is not usually a great idea. If you really have to, you should really make sure it is properly secured. You can
+find one way later in this document.
 
 Note: in the future the Admin API will also allow configuration of gateway entities such as pipeline and policy
```
Comment (altsang) [inline docs/admin/index.md]: we want to remove this as well right? no longer true - the admin API does do pipeline/policy manipulation as of now

**Review by altsang — CHANGES_REQUESTED** [review]

File: docs/admin/index.md (hunk 2 above)
Comment (XVincentX) [reply docs/admin/index.md]: Gosh, true

File: docs/admin/index.md (hunk 1 above)
Comment (XVincentX) [reply docs/admin/index.md]: Actually we do not have such thing. I'll restore it.

**Comment by XVincentX** [comment]
@DrMegavolt Please have another look when you've got time, thanks!

**Review by altsang — APPROVED** [review]

**Pull request merged**